src/main/java/seedu/address/logic/commands/AddPersonCommand.java

```diff
@@ -16,7 +16,7 @@
 /**
  * Adds a person to the address book.
  */
-public class AddCommand extends Command {
+public class AddPersonCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
```

Comment:
You can change the command word here
```suggestion
    public static final String COMMAND_WORD = "addPerson";
```

Reply:
ok will do!